CMakeLists.txt

```diff
@@ -150,16 +150,24 @@ if(FTXUI_ENABLE_INSTALL)
   install(DIRECTORY include/ftxui DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
   include(CMakePackageConfigHelpers)
+  configure_package_config_file(ftxui-config.cmake.in
+    ${CMAKE_CURRENT_BINARY_DIR}/ftxui-config.cmake
+    INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/ftxui/cmake
+    PATH_VARS CMAKE_INSTALL_INCLUDEDIR
+  )
   write_basic_package_version_file(
-    ftxui-config.cmake
+    ftxui-config-version.cmake
     VERSION ${PACKAGE_VERSION}
     COMPATIBILITY AnyNewerVersion
   )
 
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ftxui-config.cmake
+          DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/ftxui
+  )
   install(EXPORT ftxui-export
-    FILE ftxui-config.cmake
```

Comment:
Could you turn every tabs into 2 spaces?

Reply:
oops, sorry, of course.

Reply:
Thanks!